### PR TITLE
Archiver UI fixes.

### PR DESCRIPTION
**Pull request by akash1810**
Separating core styles for archiver into single file to make it more componentised. Also fixes hover state of archiver when in top-bar.

Before:
after

After:
before

### CODE REVIEW

File: kahuna/public/js/edits/ui-archiver/archiver.js [hunk 1]
```diff
@@ -35,7 +37,8 @@ archiver.controller('ArchiverCtrl', ['$scope', '$window', 'archiveService', 'onV
 archiver.directive('uiArchiver', [function() {
     return {
         restrict: 'E',
-        controller: 'ArchiverCtrl as archiver',
+        controller: 'ArchiverCtrl',
+        controllerAs: 'ctrl',
```
Comment (akash1810) [inline kahuna/public/js/edits/ui-archiver/archiver.js]: `controllerAs: 'crtl'` is more consistent with the rest of the project.

File: kahuna/public/js/edits/ui-archiver/archiver.js [hunk 2]
```diff
@@ -9,7 +11,7 @@ export var archiver = angular.module('kahuna.edits.archiver', [
 archiver.controller('ArchiverCtrl', ['$scope', '$window', 'archiveService', 'onValChange',
                     function($scope, $window, archiveService, onValChange) {
 
-    var ctrl = this;
+    let ctrl = this;
```
Comment (theefer) [inline kahuna/public/js/edits/ui-archiver/archiver.js]: `const`?

**Comment by theefer** [comment]
:+1:

File: kahuna/public/js/edits/ui-archiver/archiver.js (hunk 1 above)
Comment (jamesgorrie) [inline kahuna/public/js/edits/ui-archiver/archiver.js]: `crtl`? :confetti_ball:

Comment (akash1810) [inline kahuna/public/js/edits/ui-archiver/archiver.js]: typo... at least its in the comment, rather than the code ;)

**Pull request merged**